lib/orogen/templates/tasks/tasks.pc

```diff
@@ -32,5 +32,5 @@ Libs: <%= libs.to_a.sort.join(" ") %> -L${libdir} -l<%= project.name %>-tasks-<%
 <% if project.extended_state_support? %>
 <%   cflags << "-I${includedir}/#{project.name}/types" %>
 <% end %>
-Cflags: -I${includedir} <%= cflags.to_a.sort.join(" ") %>
+Cflags: -I${includedir} <%= cflags.to_a.join(" ").split(" ").uniq.sort.join(" ") %>
```

Comment:
Ok. Removing that part from the merge request then and looking for the source of this 'unintended' cflags format.